hs_core/templates/pages/genericresource.html

```diff
@@ -1416,6 +1416,9 @@ <h4 class="modal-title">Confirm file deletion</h4>
             </div>
             </div>
         <!-- ==== Tabs end ==== -->
+        <div class="rating">
```

Comment:
This is another fix I slated in for a regression error that was probably introduced by some conflict in merging along the way (this issue was raised in github by @Castronova). Specifically, the rating for resource code was removed (which was present on beta, but not on www), so I am adding the code back so that users can rate resources as well as rate comments to resolve that regression error. My branch commit history should give more details on this.